docs/tutorial/stacking_explanation.md

```diff
@@ -0,0 +1,13 @@
+# Stacking explanation
+
+In the following animation you can see an example how selecting different
+measures and dimensions cause different stacking behaviour in the chart.
+
+First we set the x channel to a dimension and an arbitrary constant value to
+illustrate stacking. Then by changing the constant to a measure the markers’
+width change according to the values. Finally when the dimension is removed the
+bars get ordered by the measure.
```

Comment:
What happens under the hood is that if no dimension present, then the markers measured from the origo.